src/test/java/io/aiven/kafka/connect/common/output/parquet/ParquetOutputWriterTest.java

```diff
@@ -26,11 +26,11 @@
 import java.nio.file.Path;
 import java.nio.file.StandardOpenOption;
 import java.util.ArrayList;
+import java.util.Arrays;
 import java.util.Collection;
 import java.util.Collections;
 import java.util.HashMap;
 import java.util.List;
```

Comment:
```suggestion
```

Reply:
`List` is still used on line 63. Another thing that avoids this problem is that my VSCode gives color hints for this type of unnecessary imports. Java types are awesome!